spec/requests/api/v1/npq/previous_funding/show_spec.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "NPQ Funding API", type: :request do
+  let(:token) { NPQRegistrationApiToken.create_with_random_token!(private_api_access: true) }
+  let(:bearer_token) { "Bearer #{token}" }
+  let(:parsed_response) { JSON.parse(response.body) }
+
+  let(:base_url) { "/api/v1/npq/previous_funding" }
+
+  let(:npq_course_identifier) { "npq-leading-literacy" }
+  let(:trn) { "1234567" }
+  let(:get_an_identity_id) { "87654321" }
+
+  def stub_funding_eligibility_response(trn: nil, get_an_identity_id: nil, npq_course_identifier: nil)
+    stubbed_response = {
+      "previously_funded" => SecureRandom.uuid,
+      "previously_received_targeted_funding_support" => SecureRandom.uuid,
```

Comment:
Minor but any reason this is not: `previously_received_targeted_support_funding` in line with other definitions across our system?

Reply:
No particular reason for this API to use it in particular other than to replicate the existing API's response body. Why it was phrased like this when I added it to that originally is likely down to a mistake.